src/pages/Dashboard.vue

```diff
@@ -363,5 +363,9 @@ export default {
   .inner-row {
     flex-direction: column;
   }
+
+  .hosting-preferences {
```

Comment:
Can you also add your explanation about why we're hiding the hosting-preference class for context here?